src/app.rb

```diff
@@ -63,7 +79,8 @@ def create_book
     author = gets.chomp
     book = Book.new(title, author)
     $book_list << book
```

Comment:
- [ ] I encountered an error when I tried to create a book. The error states that `undefined method `<<' for nil:NilClass (NoMethodError)` at this line. I will suggest you try to investigate it.

| Screenshot |
:-------------:|
<img width="677" alt="Screen Shot 2022-11-08 at 10 52 19 PM" src="https://user-images.githubusercontent.com/39523312/200694657-4c87cfcd-42c4-4692-b813-bb60e059769b.png"> |